examples/.env.local

```diff
@@ -1,2 +0,0 @@
-PROJECT_ID=changeme
```

Comment:
can you make a session example in the examples folder like here https://github.com/corbado/corbado-go/blob/main/examples/sessionstdlib/main.go please? we need to have a working session example.

just log in on production and get yourself a "short-term session value" (cbo_short_session cookie) and "fake it locally" so it uses the short-term session value. do not spent too much time on this if you are not done in 30 minutes for this ask me then i can help you because i already did it for go and it is a little tricky :)

Reply:
done